src/csharp/Grpc.IntegrationTesting/BenchmarkServiceGrpc.cs

```diff
@@ -313,6 +313,161 @@ protected override BenchmarkServiceClient NewInstance(ClientBaseConfiguration co
       }
     }
 
+    /// <summary>Lite client for BenchmarkService</summary>
+    public partial class BenchmarkServiceLiteClient : grpc::LiteClientBase
```

Comment:
I'm not a fan of including "Lite" in the client name. Would someone ever want both C Core and .NET Core clients?

What about only including Lite in the name if both clients are generated?